src/Keycloak.tsx

```diff
@@ -0,0 +1,108 @@
+import Keycloak from 'keycloak-js';
+import React from 'react';
+
+import { Auth, AuthContext } from '@rhoas/app-services-ui-shared';
+
+export let keycloak: Keycloak.KeycloakInstance | undefined;
+
+/**
+ * Get keycloak instance
+ *
+ * @return an initiated keycloak instance or `undefined`
+ * if keycloak isn't configured
+ *
+ */
+export const setKeycloakInstance = async () => {
+  if (!keycloak) await init();
+};
+
+/**
+ * Initiate keycloak instance.
+ *
+ * Set keycloak to undefined if
+ * keycloak isn't configured
+ *
+ */
+export const init = async () => {
+  try {
+    /* Keycloak configuration using OB authentication.
+     * Thiese params will change after sso is implemented.
+     * See https://issues.redhat.com/browse/MGDOBR-466
+     */
+    keycloak = new (Keycloak as any)({
```

Comment:
I would avoid `any` in favor of `unknown` or better specify an interface for it.